### PR TITLE
Version bump for beta release

### PR DESCRIPTION
**Pull request by mickmister**

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
     "id": "com.mattermost.mscalendar",
     "name": "Microsoft Calendar",
     "description": "Microsoft Calendar Integration",
-    "version": "0.1.0",
-    "min_server_version": "5.16.0",
+    "version": "0.1.0-beta",
+    "min_server_version": "5.24.0",
```
Comment (aaronrothschild) [inline plugin.json:6]: @mickmister This can't be 5.24...

Comment (aaronrothschild) [reply plugin.json:6]: Maybe 5.19.1+ for this beta? (KVstore was fixed)

Comment (levb) [inline plugin.json:6]: thought this was to stay 5.16 for the builds we make...

Comment (aaronrothschild) [reply plugin.json:6]: I thought there was a bug with KVStore cron jobs sor something in prior to 5.19.1 that MS cal relies upon.

Comment (levb) [reply plugin.json:6]: ^^

**Comment by aaronrothschild** [comment]
LGTM, Thanks Michael!

**Comment by aaronrothschild** [comment]
@jfrerich So, does a binary get released at the end of this process in the "release" section?

**Comment by jfrerich** [comment]
@aaronrothschild.  No, this version bump will sit in the repo as a normal commit.  The release happens by tagging a specific commit as a version as `v0.1.0-beta`.  For instance, you could actually add another commit right now (after this bump commit) and include it in the tagged release.

**Comment by jfrerich** [comment]
The release is achieved via a matterbuild command `/mb cutplugin --tag v0.1.0-beta --repo mattermost-plugin-mscalendar`

**Pull request merged**